packages/opc-header/dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-header</title>
+    <script src="opc-header.js"></script>
+    <style>
+        body {
```

Comment:
why these styles are appearing here? if they are common to add components it should be moved to common place

Reply:
As index.html  file is just for testing purpose its not actual code.

Reply:
`dist/` should not be added to the PR! Add `dist/` to `.gitignore`

Reply:
Added the `dist` folder to .gitignore